plugins/org.polarsys.capella.scenario.editor.embeddededitor/src/org/polarsys/capella/scenario/editor/embeddededitor/commands/XtextToDiagramCommands.java

```diff
@@ -303,11 +334,10 @@ private static boolean isSameMessage(EObject m, SequenceMessage seqMessage) {
     if (!(m instanceof org.polarsys.capella.scenario.editor.dsl.textualScenario.SequenceMessage)) {
       return false;
     }
-    org.polarsys.capella.scenario.editor.dsl.textualScenario.SequenceMessage message = 
-        (org.polarsys.capella.scenario.editor.dsl.textualScenario.SequenceMessage) m;
-    if (message.getSource().equals(seqMessage.getSendingEnd().getCoveredInstanceRoles().get(0).getName()) &&
-      message.getTarget().equals(seqMessage.getReceivingEnd().getCoveredInstanceRoles().get(0).getName()) &&
-      message.getName().equals(seqMessage.getName())) {
+    org.polarsys.capella.scenario.editor.dsl.textualScenario.SequenceMessage message = (org.polarsys.capella.scenario.editor.dsl.textualScenario.SequenceMessage) m;
+    if (message.getSource().equals(seqMessage.getSendingEnd().getCoveredInstanceRoles().get(0).getName())
+        && message.getTarget().equals(seqMessage.getReceivingEnd().getCoveredInstanceRoles().get(0).getName())
+        && message.getName().equals(seqMessage.getName())) {
       return true;
```

Comment:
This part seems to be not related to the commit, it alters the formatting.
Undo changes on the parts of code that are not related to this commit, or reapply capella.epf to redo the formatting.